modules/collabora_online_group/tests/src/Traits/GroupRelationTrait.php

```diff
@@ -31,7 +31,7 @@ protected function createPluginRelation(GroupTypeInterface $group_type, string $
     $entity_type_id = 'group_relationship_type';
 
     // Fallback for older versions.
-     if ($this->entityTypeManager()->getDefinition($entity_type_id, FALSE) === NULL) {
+     if (!$this->isGroupRelationShipType()) {
```

Comment:
As per the other comment, we should keep the inline condition, but add a comment:
```suggestion
     if ($this->entityTypeManager()->getDefinition($entity_type_id, FALSE) === NULL) {
       // The group module is at version *** or lower.
```